src/components/RestoreInstance/RestoreInstanceModal.tsx

```
@@ -17,14 +30,26 @@ function RestoreInstanceModal({
+        .toDate();
+      return date;
+    }
+
+    return new Date();
+  }, [earliestRestoreTime]);
+
   const restoreFormik = useFormik({
     initialValues: {
-      earliestRestoreTime: earliestRestoreTime
-        ? dayjs.utc(earliestRestoreTime)
+      earliestRestoreTime: "2024-11-16T10:43:19Z"
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)